pycti/utils/opencti_stix2.py

```diff
@@ -1207,6 +1211,18 @@ def prepare_export(
         result = []
         objects_to_get = []
         relations_to_get = []
+
+        # # Handle prepare_export specific case
+        if entity is not None and "type" in entity:
```

Comment:
Need explanation about that :)

Reply:
The goal here is to put the entity-specific code for the "prepare_export" and "process_multiple_fields" methods inside the entity file.
But I'm not very comfortable yet with contracts (interface) in python. So, this is the only method I have found so far to do it... Glad to discuss about it